src/hevm/src/EVM.hs

```diff
@@ -1963,15 +1966,13 @@ create self this xGas xValue xs newAddr initCode = do
 
         zoom (env . contracts) $ do
           oldAcc <- use (at newAddr)
```

Comment:
what is this doing exactly?

Reply:
it's fetching the account that's currently deployed at the address and overwrites with a new account but keeps the balance.